src/Download.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import fileDownload from 'js-file-download'
+
+export default function Download({ query }) {
+  const [selected, setSelected] = useState('json')
+  const options = ['json', 'csv', 'xlsx', 'tsv', 'ods']
+  const [downloading, setDownloading] = useState('')
+
+  const handleChange = (event) => {
+    setSelected(event.target.value)
+  }
+
+  const downloadData = () => {
+    setDownloading('Preparing Download')
+    fetch(
+      `http://localhost:8080/v1/download?format=${selected}`,
```

Comment:
@cotts I just noticed that this is now being hardcoded

Reply:
@anuveyatsu, Where should I include this? because the main GraphQL url is set on `index.htlm`

Reply:
`index.html` is for development purposes but the point is that you should read it from the attribute. As I understand you have data api running in 8080 so just modify `index.html` but don't commit it.